apps/console/src/features/users/components/user-profile.tsx

```diff
@@ -255,21 +255,22 @@ export const UserProfile: FunctionComponent<UserProfilePropsInterface> = (
                     if (schemaNames[0] === "name") {
                         const name = schemaNames[1] && userInfo[schemaNames[0]] &&
                             userInfo[schemaNames[0]][schemaNames[1]] && (
-                                tempProfileInfo.set(schema.name, userInfo[schemaNames[0]][schemaNames[1]])
-                            );
+                            tempProfileInfo.set(schema.name, userInfo[schemaNames[0]][schemaNames[1]])
+                        );
```

Comment:
Check indentation and line breaks

Reply:
Fixed with a later commit fe1e21e443a8ffbab52c59243e01535fff12edcf